.github/workflows/unittests.yml

```diff
@@ -69,9 +64,23 @@ jobs:
           EXTRA_DOCKER_OPTIONS="--name velox-backend-test-$GITHUB_RUN_ID --detach" NON_INTERACTIVE=ON MOUNT_MAVEN_CACHE=OFF tools/gluten-te/cbash.sh sleep 14400
       - name: Install Gluten Velox-backend spark3.2 and Run Unit test
         run: |
-          docker exec velox-backend-test-$GITHUB_RUN_ID bash -c 'cd /opt/gluten && \
-          mvn clean install -Pspark-3.2 -Pspark-ut -Pbackends-velox -Dcpp_tests=ON -Dcpp_benchmarks=ON \
-            -Dbuild_velox_from_source=ON -Dbuild_arrow=ON'
+          docker exec velox-backend-test-$GITHUB_RUN_ID bash -c 'cd /opt/gluten/ep/build-arrow/src && \
+          ./get_arrow.sh  && \
+          sed -i 's/BUILD_TESTS=OFF/BUILD_TESTS=ON/' ./build_arrow_for_velox.sh && \
```

Comment:
Why add 'sed -i' here? Not change it in build_arrow_for_velox.sh

Reply:
fixed.